### DISCUSSION TITLE
CocSearch in current buffer

### DISCUSSION THREAD

**Discussion by amgodoi** [neoclide/coc.nvim#4412] [Q&A] [answered]
Hi!

Is there any way to execute CocSearch in current buffer?

I do something like this:
:CocSearch -w wordtosearch -g path1/path2/path/myfile.java

Is there a better way?

**Comment by chemzqm** [comment, marked as answer]
It's designed for project search, there is `words` of [coc-lists](https://github.com/neoclide/coc-lists).

**Comment by amgodoi** [comment]
&#128077;
